### PR TITLE
Books Module

### PR DESCRIPTION
**Pull request by subabose6**
Automated the books module to click the books button and read the Website title

### CODE REVIEW

**Review by AmalaRamesh — APPROVED** [review]

File: src/test/java/test/hooks/Hooks.java [hunk 1]
```diff
@@ -26,13 +26,13 @@ public void beforeScenario(Scenario scenario){
...
-	@AfterStep
-	public void failureScreenshot(Scenario scenario) {
-		if(scenario.isFailed())	{
-			byte[] photo = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
-			scenario.embed(photo, "image/png","error_image");		
-		}
-		driver.close();
-		}
+//	@AfterStep
+//	public void failureScreenshot(Scenario scenario) {
```
Comment (AmalaRamesh) [inline src/test/java/test/hooks/Hooks.java:30]: why this code command

File: src/test/java/test/pageobjectmanager/PageObjectManager.java [hunk 2]
```diff
@@ -1,20 +1,28 @@
 package test.pageobjectmanager;
 
+import test.pojoclass.HomePagepojo;
 import test.pojoclass.LoginPojo;
 import test.pojoclass.SearchProductElement;
 
+
```
Comment (AmalaRamesh) [inline src/test/java/test/pageobjectmanager/PageObjectManager.java:7]: why give this space

File: src/test/java/test/hooks/Hooks.java [hunk 3]
```diff
@@ -26,13 +26,13 @@ public void beforeScenario(Scenario scenario){
...
+//		if(scenario.isFailed())	{
+//			byte[] photo = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
+//			scenario.embed(photo, "image/png","error_image");		
+//		}
+//		driver.close();
```
Comment (pugal-co) [inline src/test/java/test/hooks/Hooks.java:35]: Please remove this line

File: src/test/java/test/hooks/Hooks.java [hunk 4]
```diff
@@ -26,13 +26,13 @@ public void beforeScenario(Scenario scenario){
 		}	
 	}
 	
-	@AfterStep
-	public void failureScreenshot(Scenario scenario) {
-		if(scenario.isFailed())	{
-			byte[] photo = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
-			scenario.embed(photo, "image/png","error_image");
```
Comment (pugal-co) [inline src/test/java/test/hooks/Hooks.java:33]: Where the screenshot saves?

File: src/test/java/test/pageobjectmanager/PageObjectManager.java (hunk 2 above)
Comment (AmalaRamesh) [inline src/test/java/test/pageobjectmanager/PageObjectManager.java:7]: why give this space

**Pull request merged**